test/TestUtilities/Test.Utility/Signing/TimestampServiceOptions.cs

```diff
@@ -15,6 +16,9 @@ public sealed class TimestampServiceOptions
         public bool ReturnFailure { get; set; }
         public bool ReturnSigningCertificate { get; set; }
         public Oid SignatureHashAlgorithm { get; set; }
+        public DateTimeOffset? IssuedCertificateNotBefore { get; set; }
+        public DateTimeOffset? IssuedCertificateNotAfter { get; set; }
+        public DateTime? GeneralizedTime { get; set; }
```

Comment:
Seems like this should be `DateTimeOffset?` as well.  It's OK if internally `TimestampService` converts to satisfy BouncyCastle.

Reply:
Yeah, I made it `DateTime` instead of `DateTimeOffset` because BouncyCastle needs it as `DateTime` and I wanted to avoid converting it. Should I add a comment specifying to use UTC or local time then?

Reply:
I would use `DateTimeOffset?` here and then convert internally.  Then it's unambiguous what time zone to use and you don't need a comment.